scripts/2022_05_12_migrate_metapool_rewards.py

```diff
@@ -0,0 +1,99 @@
+from helpers import CHAIN_IDS, LP_MIGRATOR_ADDRESSES, MINICHEF_ADDRESSES, MULTISIG_ADDRESSES, DISPERSE_APP_ADDRESSES
+from ape_safe import ApeSafe
+from brownie import accounts, network, web3, utils
+from eth_utils import to_wei
+
+from scripts.utils import confirm_posting_transaction
+
+
+def main():
+    """Change Minichef rewards from old pools to new pools. Add the new pools to the migrator contract."""
+
+    print(f"You are using the '{network.show_active()}' network")
+    deployer = accounts.load("deployer")  # prompts for password
+    multisig = ApeSafe(MULTISIG_ADDRESSES[CHAIN_IDS["MAINNET"]])
+
+    minichef = multisig.contract(MINICHEF_ADDRESSES[CHAIN_IDS["MAINNET"]])
+    migrator = multisig.contract(LP_MIGRATOR_ADDRESSES[CHAIN_IDS["MAINNET"]])
+
+    # Set rewards to oudated pool lp tokens to 0
+    # outdated tbtc meta pool
+    minichef.set(5, 0, "0x0000000000000000000000000000000000000000", False)
+    # outdated susd meta pool
+    minichef.set(6, 0, "0x0000000000000000000000000000000000000000", False)
+    # outdated wcusd meta pool
+    minichef.set(7, 0, "0x0000000000000000000000000000000000000000", False)
+
+    # Add new rewards for the meta pools
+    # tbtc v3 meta pool lp token
+    minichef.add(25, "0xA2E81Eb93F0F9814ae9A3bea2D2A63408f2709C1",
+                 "0x0000000000000000000000000000000000000000")
+    # susd v3 meta pool lp token
+    minichef.add(50, "0x444F94460a641429CDa4e38E02E51642Cc38276A",
+                 "0x0000000000000000000000000000000000000000")
+    # wcusd v3 meta pool lp token
+    minichef.add(2, "0x0dB8b09c13FE21913faF463274cE8e0a51719f16",
```

Comment:
should this be so low?

Reply:
✅  address

Reply:
![image](https://user-images.githubusercontent.com/4276174/168660860-6db79dd7-92dc-4473-bb24-8b039041afa5.png)
Yep allocpoint is set to 2 for wcusd pool